api/scripts/wazuh-apid.py

```diff
@@ -213,9 +213,12 @@ def set_logging(log_path=f'{API_LOG_PATH}.log', foreground_mode=False, debug_mod
     log_path = f'{API_LOG_PATH}.log'
     foreground_mode = args.foreground
```

Comment:
This is already defined in each of the calls to `set_logging`.
```suggestion
```

Reply:
Deleted in commit d35012ab2665a1f75b9e47a67c7354eab8141573.